src/app/code/community/Omise/Gateway/Block/Form/Offsiteinternetbankingpayment.php

```diff
@@ -30,4 +30,5 @@ protected function _toHtml()
 
         return parent::_toHtml();
     }
+    
```

Comment:
and that enter... probably unecessary

Reply:
Matches the one at the top

Reply:
In that case, maybe remove one at the top would be better as it's more consistent with the rest of the code (that one at the top might be my fault).